### DISCUSSION TITLE
sqoop拉取数据到hdfs时报权限错误

### DISCUSSION THREAD

**Discussion by ChanKamShing** [apache/dolphinscheduler#6470] [Q&A]
![image](https://user-images.githubusercontent.com/48425984/136646119-0b74563b-907b-48c0-85f6-1f72b7d72cd6.png)
有没有知道需要怎么设置？

**Comment by sakinsun** [comment]
你运行ds的时候用户是 dschudler不是 root。所以你要在ds里面设置一个新的租户，租户名字和代码都是 ：root ，然后在编辑ds模块保存的时候，租户 选择root。就行了。
